src/document/README.md

```diff
@@ -6,41 +6,51 @@ Metaschema in, documentation out.
 
-### `mvn-schemadocs-html-xpl.sh`
+### `mvn-schemadocs-testsite-xpl.sh`
 
 Produces a set of interlinked HTML files documenting the models (XML and object/JSON) for a metaschema, in a subdirectory.
 
+Arguments are provided to the script as `METASCHEMA_XML OUTPUT_DIR SCHEMA_NAME`, for example
+
 ```
-> ./mvn-schemadocs-html-xpl.sh a-metaschema.xml metaschemaA docs METASCHEMA_XML SCHEMA_NAME OUTPUT_DIR
+> ./mvn-schemadocs-testsite-xpl.sh a-metaschema.xml docs metaschemaA
 ```
 
 where
 
 - the script `./mvn-schemadocs-html-xpl.sh` is available (in the directory or on the system path)
 - `a-metaschema.xml` is a metaschema (top-level module) as a relative path (file URI)
-- `metaschemaA` is the label to use in the documentation produced (file names and links)
 - `docs` is a *relative* path (URI syntax) for writing serialized outputs (HTML files)
+- `metaschemaA` is the label to use in the documentation produced (file names and links)
 
 Assuming the incoming metaschema is valid, correct, and correctly linked (imports resolved), HTML file outputs are written to the indicated subdirectory.
 
-The result file names are defined in the underlying XProc, `METASCHEMA-HTML-DOCS.xpl`.
+The result file names are defined in the underlying XProc, `METASCHEMA-DOCS-TESTSITE-write.xpl`.
 
-### `mvn-schemadocs-debug-xpl.sh`
+## Pipelines
 
-This script invokes the base 'traceable' XProc, `METASCHEMA-DOCS-TRACE.xpl`. By managing exposed ports (now binding to `/dev/null` for intermediate results and file paths for HTML results) through this script, intermediate and final outputs can be examined and assessed. Use for debugging.
+All these pipelines have a primary input source port named `METASCHEMA`, which should be provided with a valid metaschema whose imports are resolvable and valid.
 
-## Pipelines
+### `METASCHEMA-DOCS-DIVS-write.xpl`
 
-### `METASCHEMA-HTML-DOCS.xpl`
+Given a `path` to write to and a key name (schema name), this pipeline serializes and writes a set of documentation rooted at HTML `div` elements, suitable for ingestion into Hugo or any other HTML-based publishing system.
 
-This 'wrapper' pipeline includes the base pipeline `METASCHEMA-DOCS-TRACE.xpl` as a defined step, and invokes it with metaschema input while configuring its runtime. Consequently it is much simpler to run.
+NB: Markdown can be acquired for docs by reducing this HTML to Markdown. Make inquiries if this is of use.
 
-Use this pipeline with metaschema input when you want HTML file results written to a specific location by the XProc.
+### `METASCHEMA-DOCS-DIVS.xpl`
 
-### `METASCHEMA-DOCS-TRACE.xpl`
+For ease of configurability, this pipeline works like `METASCHEMA-DOCS-TESTSITE-write.xpl` except Given a `path` to write to and a key name (schema name), this pipeline serializes and writes a set of documentation rooted at HTML `div` elements, suitable for ingestion into Hugo or any other HTML-based publishing system.
```

Comment:
Something doesn't look right.

1. "except Given" looks like the first sentence is missing its ending.
2. Is the sentence starting with "Given" supposed to be here?
3. Is the first sentence supposed to compare this pipeline to `METASCHEMA-DOCS-DIVS-write.xpl` (the one above) or `METASCHEMA-DOCS-TESTSITE-write.xpl` (the one below, which refers back to this one)?